magellan-library/src/main/java/com/wealthfront/magellan/transitions/CircularRevealTransition.kt

```diff
@@ -31,33 +37,38 @@ public class CircularRevealTransition(private val clickedView: View) : MagellanT
     direction: Direction,
     onAnimationEndCallback: () -> Unit
   ) {
-    val clickedViewCenter = getCenterClickedView(from as ViewGroup)
-    val circularRevealCenterX = clickedViewCenter[0]
-    val circularRevealCenterY = clickedViewCenter[1]
-    val finalRadius = hypot(to.width.toDouble(), to.height.toDouble()).toFloat()
-    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
-      animator = ViewAnimationUtils.createCircularReveal(
-        to, circularRevealCenterX,
-        circularRevealCenterY, 0f, finalRadius
-      ).apply {
-        addListener(object : AnimatorListenerAdapter() {
-          override fun onAnimationEnd(animation: Animator) {
-            animator = null
-            onAnimationEndCallback()
-          }
-        })
-        interpolator = FastOutSlowInInterpolator()
-      }
-      animator!!.start()
-    } else {
+    if (Build.VERSION.SDK_INT < Build.VERSION_CODES.LOLLIPOP) {
       onAnimationEndCallback()
+      return
     }
+
+    val clickedViewContainer = if (direction == Direction.FORWARD) from else to
```

Comment:
Support back animation properly. We're assuming that the clicked view is inside the "from" container if we're going forward, or inside the "to" container if we're going backward.